Src/library/docking/implementation/configuration/sd_open_config_mediator.e

```diff
@@ -809,30 +810,33 @@ feature {NONE} -- Implementation
 			loop
 				l_data := a_tool_bar_data.item
 				check is_floating_tool_bar_data: l_data.is_floating end
-				l_title := l_data.title
-				check l_title /= Void end -- Implied by check `is_floating_tool_bar_data'
-				l_content := l_tool_bar_manager.content_by_title (l_title)
+				if attached l_data.title as l_title then
```

Comment:
Wouldn't it better to have `content_by_title` return Void if no good title is provided? I looked at the code and we would end up in a crash if ever in the loop we cannot set Result.